portal_tables/unify_grant_tables.py

```diff
@@ -0,0 +1,145 @@
+"""Combine Synapse projects that are separated 
+by grant into the CCKP admin project table.
+
+This script assumes the folders for PublicationView,
+DatasetView, and ToolView, respectively, remain 
+consistent in the CCKP admin project table.
+
+author: victor.baham
+"""
+
+# command line arguments: 1. Portal - Grants Merged Synapse ID 2. CCKP Admin Project Synapse ID
+# test value for argument 1: syn42801895
+# test value for argument 2: syn35629947 
+
+import argparse
+import synapseclient
+from synapseclient import Project, File, Folder
+from synapseclient import Schema, Column, Table, Row, RowSet, as_table_columns
+import itertools
+import pandas as pd
+
+def get_args():
+    #Retrieve Synapse table IDs from command line
```

Comment:
When describing the overall function, it's better to use a docstring rather than a comment.

That way, if/when you go to create a docs page for your script, the tool will know where the function description is!